.github/workflows/publish.yaml

```diff
@@ -0,0 +1,89 @@
+name: Publish Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: 'Version of the latest release (e.g. v0.42.0)'
+        required: false
+        default: ''
+  repository_dispatch:
+    types: [ocm-cli-release]
+
+jobs:
+
+  push-to-aur:
+    name: Update Arch Linux User Repository
+    runs-on: ubuntu-latest
+    steps:
+    - name: Ensure proper version
+      run: |
+        if [ -z "${{ github.event.client_payload.version }}" ]; then
+          if [ -z "${{ github.event.inputs.version }}" ]; then
+            echo "Version not provided"
+            exit 1
+          fi
+          echo "RELEASE_VERSION=$(echo ${{ github.event.inputs.version }} | tr -d ['v'])" >> $GITHUB_ENV
+        else
+          echo "RELEASE_VERSION=$(echo ${{ github.event.client_payload.version }} | tr -d ['v'])" >> $GITHUB_ENV
+        fi
+        cat $GITHUB_ENV
```

Comment:
Is this command required or for debugging purposes?

Reply:
yes and no... since the workflow can be triggered in multiple different ways, I wanted to have the actual used version appearing somewhere in the logs.